sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -120,4 +121,16 @@ export class UserController extends BaseController {
       throw error;
     }
   }
+
+  @Get("/institutionUser")
```

Comment:
IMO, the information being returned here is not institution-specific. I would recommend that this endpoint would be generic GET users.

Reply:
I guess we decided to have it as /institution. Right @dheepak-aot, @andrewsignori-aot @ann-aot  ?

Reply:
Yes @guru-aot